AutoLayoutHelper/UIView+AutoLayoutHelper.swift

```diff
@@ -45,12 +45,12 @@ extension UIView {
     
     :returns: An array of 4 x NSLayoutConstraint objects (top, left, bottom , right) if the superview exists (otherwise an empty array)
     */
-    func fillSuperView(edges: UIEdgeInsets) -> [NSLayoutConstraint] {
+    func fillSuperView(edges: UIEdgeInsets = UIEdgeInsetsZero) -> [NSLayoutConstraint] {
```

Comment:
Do you think it would be good to add default parameters' description to the documentation for this method (and other methods that got default params in this PR)?